#goal definition
int32 seconds     #Runtime. 0 for indefinite.
float32 max_vel_x #Maximum translation velocity.
float32 max_rot_vel #Maximum rotation velocity.
float32 max_dist  #Maximum distance of detected humans. Everything above will be ignored and the max_vel_x value will be used.
float32 min_dist  #The distance to a human at which the robot will stop (have 0.0 velocity).
float32 time_to_reset #The time it takes while no human is detected to resume to 'max_vel_x' speed.
---
#result definition
bool expired #Returns true when the time 'seconds' is up.
---
#feedback
int32 num_found_humans #Number of currently detected humans.
float64 min_dist       #The distance of the closest human to the robot.
float32 current_speed  #The current speed of the robot.
float32 current_rot    #The current rotaional speed of the robot.
float32 remaining_runtime #The remaining run time. -1 if indefinite.
float32 time_to_reset  #The time until the speed will be reset to 'max_vel_x' (see 'threshold')
